internal/commands/all/all.go

```diff
@@ -138,6 +138,7 @@ func BuildCommands(rootCmd *cobra.Command, conf *config.Config) {
 
 	// Databases
 	databaseCommand := commands.BuildCommand(database.BaseDatabaseCommand(), rootCmd, conf)
+	commands.BuildCommand(database.CreateCommand(), databaseCommand.Cobra(), conf)
```

Comment:
Because of #393 this change would need to be in `internal/commands/base/base.go` instead.